rules.mk.j2

```diff
@@ -220,17 +221,19 @@ else
 	ci/vm.sh network_test | tee build/$@.log >/dev/$T
 endif
 
+build/osie-aarch64.tar.gz: /proc/sys/fs/binfmt_misc/qemu-aarch64
 build/osie-aarch64.tar.gz: SED=/FROM/ s|.*|FROM multiarch/ubuntu-debootstrap:arm64-xenial|
-build/osie-x86_64.tar.gz: SED=
+build/osie-x86_64.tar.gz:  SED=
```

Comment:
extra space?

Reply:
was on purpose for alignment ;p. Doesn't hurt.